d3a_interface/constants_limits.py

```diff
@@ -46,8 +47,8 @@ class GeneralSettings:
         # Boolean flag which forces d3a to dispatch events via redis channels
         EVENT_DISPATCHING_VIA_REDIS = False
 
-        RATE_DECREASE_PER_UPDATE_RANGE = RangeLimit(0, 1000)
-        RATE_CHANGE_PER_UPDATE = RangeLimit(0, 1000)
+        RATE_DECREASE_PER_UPDATE_RANGE = RateRange(0, 1000)
+        RATE_CHANGE_PER_UPDATE = RateRange(0, 1000)
```

Comment:
Isn't this a range too ? Then why not being consistent with the naming above and renaming it to RATE_CHANGE_PER_UPDATE_RANGE ?

Reply:
Thanks for the catch, `RATE_CHANGE_PER_UPDATE ` isn't used anymore. If you agree, I will remove it.

Reply:
Sorry, Im keeping/updating `RATE_CHANGE_PER_UPDATE_RANGE`, as it looks more generic to me compared to `RATE_DECREASE_PER_UPDATE_RANGE `